### PR TITLE
Adding Ikea E1812

### PR DESCRIPTION
**Pull request by joshua-schnabel**

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -2647,6 +2647,30 @@ const devices = [
...
     },
+    {
+        zigbeeModel: ['TRADFRI SHORTCUT Button'],
+        model: 'E1812',
+        vendor: 'IKEA',
+        description: 'TRADFRI Shortcut Button',
+        supports: 'on, brightness up/stop',
+        fromZigbee: [
+            fz.command_on, fz.legacy_genOnOff_cmdOn, fz.command_move,
+            fz.legacy_E1743_brightness_up, fz.command_stop, fz.legacy_E1743_brightness_stop, fz.battery,
```
Comment (Koenkk) [inline devices.js:2658]: Please use the non-legacy versions, so for e.g. `legacy_E1743_brightness_up` -> `command_move`

File: devices.js [hunk 2]
```diff
@@ -2647,6 +2647,30 @@ const devices = [
...
+        ],
+        exposes: [e.battery(), e.action(['on', 'brightness_move_up', 'brightness_stop'])],
+        toZigbee: [],
+        // OTA is not supported at the moment
+        // ota: ota.tradfri,
```
Comment (Koenkk) [inline devices.js:2663]: I think we can enable this as I expect ikea to update their image index soon with this.

**Comment by Koenkk** [comment]
Note to self: after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/448

File: devices.js (hunk 1 above)
Comment (joshua-schnabel) [reply devices.js]: I have removed the "legacy" entries. Is that what you meant?

**Comment by Koenkk** [comment]
Thanks! Closes https://github.com/Koenkk/zigbee2mqtt/issues/4789

**Pull request merged**